source/Timezones.mc

```diff
@@ -0,0 +1,114 @@
+using Toybox.WatchUi as Ui;
+using Toybox.System as Sys;
+using Toybox.Application as App;
+using Toybox.Graphics as Gfx;
+using Toybox.Time.Gregorian as Calendar;
+
+(:timezones)
+class Timezones extends Ui.Drawable {
+
+	var utcOffset = new Time.Duration(-Sys.getClockTime().timeZoneOffset);
+	
+	var tzLocations = [];
+	var UTC_Offsets = {};
+	var mTimeZones;
+	
+	var mhalfDCWidth; 
+	
+	var mTopY;
+	var mBottomY;
+
+    var faceWidth;
+    var faceHeight;
+
+	function initialize(params) {
+		Drawable.initialize(params);
+
+		mTopY = params[:topY];
+		mBottomY = params[:bottomY];
+		
+		setupTimezones();
+	}
+
+	function onSettingsChanged() {
+		setupTimezones();
+	}
+
+    function setupTimezones() {
+		tzLocations = [];
+		UTC_Offsets = {};
+        mTimeZones = App.getApp().getProperty("timezones").toNumber();
+        for (var i = 1; i <= mTimeZones; i++) {
+		    var name = App.getApp().getProperty("tz"+i+"_name");
+		    var offset = App.getApp().getProperty("tz"+i+"_offset").toNumber();
+		    var dst = App.getApp().getProperty("tz"+i+"_dst");
+		    if (name.equals("")) {
+			    continue;
+		    } else {
+			    tzLocations.add(name);
+		        System.println("Name:<"+name+">");	
```

Comment:
`println` calls should be commented out in release code, as the statically-stored string otherwise consumes memory.